miio/huizuo.py

```diff
@@ -34,44 +120,102 @@ class HuizuoException(DeviceException):
 
 class HuizuoStatus:
     def __init__(self, data: Dict[str, Any]) -> None:
-        """
-        Response of a Huizuo Pisces For Bedroom (huayi.light.pis123)
-         {'id': 1, 'result': [
-             {'did': '', 'siid': 2, 'piid': 1, 'code': 0, 'value': False},
-             {'did': '', 'siid': 2, 'piid': 2, 'code': 0, 'value': 94},
-             {'did': '', 'siid': 2, 'piid': 3, 'code': 0, 'value': 6400}
-             ]
-        }
-
-        Explanation (line-by-line):
-            power = '{"siid":2,"piid":1}' values = true,false
-            brightless(%) = '{"siid":2,"piid":2}' values = 1-100
-            color temperature(Kelvin) = '{"siid":2,"piid":3}' values = 3000-6400
-        """
-
         self.data = data
 
     @property
     def is_on(self) -> bool:
         """Return True if device is on."""
-        return self.data["power"]
+        if "power" in self.data:
+            return self.data["power"]
+        return None
 
     @property
     def brightness(self) -> int:
         """Return current brightness."""
-        return self.data["brightness"]
+        if "brightness" in self.data:
+            return self.data["brightness"]
+        return None
```

Comment:
If the attribute is optional please use `Optional[int]` as type of the return value.

Reply:
Added